doc/classes/Control.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Control" inherits="CanvasItem" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Base node for all User Interface components.
+		All User Interface nodes inherit from Control. A Control anchors anywhere within its parent node's bounding box. It features margins relative to the anchor's position, that update automatically.
 	</brief_description>
 	<description>
-		The base class Node for all User Interface components. Every UI node inherits from it. Any scene or portion of a scene tree composed of Control nodes is a User Interface.
-		Controls use anchors and margins to place themselves relative to their parent. They adapt automatically when their parent or the screen size changes. To build flexible UIs, use built-in [Container] nodes or create your own.
-		Anchors work by defining which margin do they follow, and a value relative to it. Allowed anchoring modes are ANCHOR_BEGIN, where the margin is relative to the top or left margins of the parent (in pixels), ANCHOR_END for the right and bottom margins of the parent and ANCHOR_RATIO, which is a ratio from 0 to 1 in the parent range.
-		Godot sends Input events to the root node first, via [method Node._input]. The method distributes it through the node tree and delivers the input events to the node under the mouse cursor or on focus with the keyboard. To do so, it calls [method MainLoop._input_event]. No need to enable [method Node.set_process_input] on Controls to receive input events. Call [method accept_event] to ensure no other node receives the event, not even [method Node._unhandled_input].
-		Only the one Control node in focus receives keyboard events. To do so, the Control must get the focus mode with [method set_focus_mode]. It loses focus when another Control gets it, or if the current Control in focus is hidden.
-		You'll sometimes want Controls to ignore mouse or touch events. For example, if you place an icon on top of a button. Call [method set_ignore_mouse] for that.
-		[Theme] resources change the Control's appearance. If you change the [Theme] on a parent Control node, it will propagate to all of its children. You can override parts of the theme on each Control with the add_*_override methods, like [method add_font_override]. You can also override the theme from the editor.
+		Base class for all User Interface or [i]UI[/i] related nodes. [code]Control[/code] feature a bounding rectangle that defines its extents, an anchor position relative to its parent and margins that are relative to the anchor. The margins update automatically when the node, any of its parents or the screen size change.
```

Comment:
Its more like the anchors adapt relatively to the parent, while the margins always stay constant. (They are like a fixed offset relative to each anchor position)

Reply:
Yes but as a user, you'll see the margin values update, and nothing else. In some cases they won't change (anchor to full rect and resize the parent container), but most of the time they do. Changing the phrasing